future-apps/payroll/test/contracts/Payroll_modify_employee.test.js

```diff
@@ -54,17 +54,34 @@ contract('Payroll employees modification', ([owner, employee, anotherEmployee, a
                 assert.equal(salary.toString(), newSalary, 'accrued value does not match')
               })
 
-              it('adds previous owed salary to the accrued value', async () => {
+              it('adds previous owed salary to the accrued salary', async () => {
                 await payroll.mockAddTimestamp(ONE_MONTH)
 
+                const receipt = await payroll.setEmployeeSalary(employeeId, newSalary, { from })
```

Comment:
Super small thing, but I don't think this needs to be done for this test

Reply:
I'm not sure I follow.. I was doing it to get some accrued salary added

Reply:
Ahh that makes sense; I was thinking you just wanted to change the salary 🤦‍♂️